cypress/e2e/cloud/home.test.ts

```diff
@@ -0,0 +1,238 @@
+import {Organization} from '../../../src/types'
+
+describe('Home Page Tests', () => {
+  beforeEach(() => {
+    cy.flush()
+
+    cy.signin().then(() => {
+      cy.get('@org').then(() => {
+        cy.getByTestID('home-page--header').should('be.visible')
+        cy.window().then(w => {
+          w.influx.set('alertsStatuses', true)
+          w.influx.set('notebooks', true)
+        })
+
+        cy.wait(1000)
+      })
+    })
+  })
+
+  it('should load empty Alerts Statuses dashboard', () => {
+    cy.getByTestID('alerts-statuses')
+      .scrollIntoView()
+      .should('be.visible')
+    cy.getByTestID('alerts-statuses-table-container')
+      .find('.event-row')
+      .should('have.length', 0)
+  })
+
+  it('should populate and load populated Alerts Statuses dashboard', () => {
+    cy.getByTestID('alerts-statuses')
+      .scrollIntoView()
+      .should('be.visible')
+    cy.getByTestID('alerts-statuses-table-container')
+      .find('.event-row')
+      .should('have.length', 0)
+
+    createChecks()
+    exportMockAlertsStatuses()
+
+    cy.visit('/')
+    cy.getByTestID('alerts-statuses-table-container')
+      .find('.event-row')
+      .should('have.length', 2)
+  })
+})
+
+const exportMockAlertsStatuses = () => {
+  const script1 = `import "influxdata/influxdb/v1"
+import "influxdata/influxdb/monitor"
+
+from(bucket: "_tasks")
```

Comment:
we used notebooks to generate and validate this query, but you dont have to put them all in different panels now that you have a working query.